moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -135,6 +211,14 @@ void MotionPlanningFrame::computePlanButtonClicked()
   Q_EMIT planningFinished();
 }
 
+void MotionPlanningFrame::computePlanButtonClicked()
+{
+  if (ui_->cartesian_path->checkState())
```

Comment:
I think it would make sense to call the checkbox "use_cartesian_path", to improve readibility in places like this.

Reply:
renamed